registrations/models.py

```diff
@@ -345,6 +350,30 @@ def signups_count(self):
     def attending_signups(self):
         return self.signups.filter(attendee_status=SignUp.AttendeeStatus.ATTENDING)
 
+    @transaction.atomic
+    def anonymize(self):
+        faker = Faker("fi_FI")
```

Comment:
There might be some business or other reason I'm not aware of, but in general in my opinion it would be better to use values that cannot be mistaken for real values, like `<DELETED>` or something.

Reply:
I don't see any business reason to have fake data. I changed it to `<DELETED>` to make it easier to see that real data is anonymized.